F/FFMPEG/FFMPEG/build_tarballs.jl

```diff
@@ -23,22 +23,22 @@ products = [
 dependencies = [
     HostBuildDependency("YASM_jll"),
     BuildDependency("nv_codec_headers_jll"),
-    Dependency("libass_jll"; compat="0.15.1"),
+    Dependency("libass_jll"; compat="0.15.2"),
     Dependency("libfdk_aac_jll"; compat="2.0.3"),
     Dependency("FriBidi_jll"),
-    Dependency("FreeType2_jll"; compat="2.10.4"),
+    Dependency("FreeType2_jll"; compat="2.13.3"),
     Dependency("LAME_jll"),
     Dependency("libvorbis_jll"),
     Dependency("libaom_jll"),
     Dependency("Ogg_jll"),
     BuildDependency("LibVPX_jll"), # We use the static archive
     Dependency("x264_jll"; compat="10164.0.0"),
```

Comment:
We're a long way from having all deps for riscv64 built, but we've got this one: #10200
```suggestion
    Dependency("x264_jll"; compat="10164.0.1"),
```

Reply:
The most basic thing missing for riscv64 is OpenSSL...

Reply:
Done.